easy_my_coop_website/controllers/main.py

```diff
@@ -256,29 +256,30 @@ def validation(self, kwargs, logged, values, post_file):
             redirect = "easy_my_coop_website.becomecompanycooperator"
             email = kwargs.get("company_email")
 
-        if (
-            "g-recaptcha-response" not in kwargs
-            or kwargs["g-recaptcha-response"] == ""
-        ):
-            values = self.fill_values(values, is_company, logged)
-            values.update(kwargs)
-            values["error_msg"] = _(
-                "the captcha has not been validated,"
-                " please fill in the captcha"
-            )
+        if request.website.company_id.captcha_type == 'google':
```

Comment:
@houssine78 Better now? A selection field in company. Now it's 'google' or 'none' but in the future it can be set to other captcha library.

Reply:
Hi @cesarlr, Houssine is on holiday for two weeks. The ideal way would be to check the recatcha in a separate function that could be overloaded in a new module implementing the new recaptcha method.

For now, you can just leave a comment suggesting this and I'll approve the PR.